DailySelfie/app/src/main/java/com/teamagam/dailyselfie/DailyAlarmReceiver.java

```diff
@@ -0,0 +1,49 @@
+package com.teamagam.dailyselfie;
+
+import android.app.Notification;
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+
+
+public class DailyAlarmReceiver extends BroadcastReceiver{
+
+    private final int DAILY_NOTIFICATION_ID = 0;
+    private Context mContext;
+
+    @Override
+    public void onReceive(Context context, Intent intent) {
+        mContext = context;
```

Comment:
I indeed had doubts. But as you said, I thought it is safe. I'll change it back...